src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml

```diff
@@ -1,6 +1,7 @@
 <UserControl x:Class="CalculatorApp.GraphingCalculator"
              xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:Windows10version1903="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract, 8)"
```

Comment:
We should not associate UniversalApiContract and windows versions and be consistent with other namespaces.

`xmlns:contract8Present="..."`

Reply:
Is there a reason we shouldn’t or is it just for consistency? It is what VS auto-inserted for me, should we send feedback for them to change that?